sktime/forecasting/base/adapters/_generalised_statsforecast.py

```diff
@@ -254,3 +271,197 @@ def _predict_interval(self, fh, X, coverage):
         final_interval_predictions = pandas.concat(interval_predictions, copy=False)
 
         return final_interval_predictions
+
+
+class StatsForecastBackAdapter(_TS):
+    """StatsForecast Back Adapter.
+
+    StatsForecastBackAdapter is a wrapper for sktime forecasters to be used in
+    StatsForecast composite models.
+
+    Parameters
+    ----------
+    estimator : sktime estimator
```

Comment:
```suggestion
    estimator : sktime forecaster
```

Non-forecaster estimators will not work here.